src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -442,3 +408,11 @@ This can happen for a couple of reasons:
 	}
 	return err
 }
+
+func StringPtr(v string) *string {
+	return &v
+}
+
+func BoolPtr(v bool) *bool {
```

Comment:
Since this is only used in the podman client, this may be worth moving to that file itself and not exporting it.